web/client/epics/context.js

```diff
@@ -38,7 +45,19 @@ const createContextFlow = (id, action$, getState) =>
         ? getResource(id)
             // TODO: setContext should put in ConfigUtils some variables
             // TODO: solve the problem of initial state used to configure plugins partially
-            .switchMap((resource) => Observable.of(setResource(resource), setContext(resource.data)))
+            .switchMap(resource => {
+                const templates = get(resource, 'data.templates', []);
+                return Observable.forkJoin(
+                    Observable.of(resource),
+                    ...templates.map(({id: templateId}) =>
+                        getResource(templateId).switchMap(templateResource => Observable.of({
```

Comment:
This could slow down a lot initial loading: 
 - [ ] Postpone template details to when the user opens the templates list
- [ ] data should be loaded only for the template selected, when selected. (you can use getResource options to avoid data fetch on start and do only on effective selection).
 - we are looking for a solution to let you get the list with a single ajax call. About this please wait a feedback from @tdipisa 
- [ ] We are looking for a solution that allow you to load all the list in